scripts/seedsDB.js

```diff
@@ -58,49 +58,49 @@ db.Project.remove({})
   // This file empties the risks collection and inserts the books below
   const riskSeed = [   
     {
-        title: "title1",
+        title: "London Eye",
         riskId: "riskId1",
-        description:"discription1",
+        description: "Annual maintenance",
```

Comment:
Needs to be a risk, rather than a project description. E.g. "Risk of structural failure of spokes on London eye during maintenance"